mimic/model/maas_objects.py

```diff
@@ -0,0 +1,229 @@
+"""
+MaaS API data model
+"""
+
+import random
+
+
+def random_letters(len_min, len_max):
+    """
+    Makes a string with random letters (a-z).
+    """
+    return ''.join([chr(ord('a') + random.randint(0, 25))
+                    for _ in xrange(random.randint(len_min, len_max))])
+
+
+class Metric(object):
+    """
+    Models a MaaS metric type.
+    """
+    def __init__(self, name, type, clock, unit='other'):
+        """
+        Initializes the metric.
+        """
+        self.name = name
```

Comment:
Not a blocking comment, but, for simple objects like this, you should probably be using `@characteristic.attributes` to declare it; objects declared with `characteristic` get a bunch of things for free:

- a nice `repr`
- comparison operators so that instances can be sorted
- hashing operators (which, in this case, would make this object *not* hashable because it contains a dictionary: which is what you want)
- an API for enumerating attributes
- optional metadata for types of attributes

For more detail on the differences, see http://characteristic.readthedocs.org/en/stable/why.html#hand-written-classes

There's no *specific* reason that a `Metric` absolutely needs any of these features, but it's helpful if we consistently do this throughout the codebase, so that more objects provide these features.